repository-data/application/src/main/resources/hcm-content/content/documents/administration/publication-system/headers.yaml

```diff
@@ -133,6 +154,9 @@
     - headers.geographical-granularity
     - headers.publication-date
     - headers.next-publication-date
+    - headers.publication-label
```

Comment:
maybe should be in a separate file - `administration/publication-system/labels.yaml`